### ISSUE TITLE
Displays the whole result set instead of just the number of items per page

### ISSUE THREAD

**Issue by bramley** [Krakengr/search-plugin-bludit#1]
This comparison is the wrong way around. Say the number of items is 6 and items per page is 4, then the whole result set is displayed with next/previous navigation links

https://github.com/Krakengr/search-plugin-bludit/blob/8d3270de285208c27431640b394682923e172260/db-search/plugin.php#L224

Reversing the comparison gives the expected result

			if( $itemsPerPage >= $this->numberOfItems )

**Comment by Krakengr** [comment]
The purpose of this comparison is to split the results according to the number of items per page and when the number of items is greater than this number. Say the number of items is 20 and the number of items per page is 15, then the result set will be split into two pages, but if the number of items is 14 and the number of items per page is 15, then the result set will be displayed as it is without any navigation link. I can't understand where the error is.

**Comment by bramley** [comment]
I agree that is what is intended, but the condition `<=` should be` >=`

When `$itemsPerPage` is less than or equal to `$this->numberOfItems` (in your example 15 <= 20) then the whole list is being assigned to `$this->pagesFound`. The intention is to split into two pages.

**Comment by Krakengr** [comment]
I'll try it asap.
